From cb3e4a8eeaa0b7f945e742ecce9894aae3070678 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 21 Jul 2020 09:11:17 -0700
Subject: [PATCH 1449/1851] v4l: xilinx: dprx: Remove unused variables

commit c12449e1895f9e20dca5bf930914a387c243196a from
https://github.com/Xilinx/linux-xlnx.git

drivers/media/platform/xilinx/xilinx-dprxss.c: In function 'xdprxss_get_stream_properties':
drivers/media/platform/xilinx/xilinx-dprxss.c:449:5: warning: variable 'rxmsa_bpc' set but not used [-Wunused-but-set-variable]
  449 |  u8 rxmsa_bpc, pixel_width, fmt;
      |     ^~~~~~~~~
drivers/media/platform/xilinx/xilinx-dprxss.c:446:18: warning: unused variable 'bpc' [-Wunused-variable]
  446 |  static const u8 bpc[] = { 6, 8, 10, 12, 16 };
      |                  ^~~

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Rajesh Gugulothu <gugulothu.rajesh@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-dprxss.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-dprxss.c b/drivers/media/platform/xilinx/xilinx-dprxss.c
index c845be9c13c1..1e8cb2c8a31c 100644
--- a/drivers/media/platform/xilinx/xilinx-dprxss.c
+++ b/drivers/media/platform/xilinx/xilinx-dprxss.c
@@ -444,10 +444,9 @@ static int xdprxss_get_stream_properties(struct xdprxss_state *state)
 	struct v4l2_mbus_framefmt *format = &state->format;
 	struct v4l2_bt_timings *bt = &state->detected_timings.bt;
 
-	static const u8 bpc[] = { 6, 8, 10, 12, 16 };
 	u32 rxmsa_mvid, rxmsa_nvid, rxmsa_misc, recv_clk_freq, linkrate;
 	u16 vres_total, hres_total, framerate, lanecount;
-	u8 rxmsa_bpc, pixel_width, fmt;
+	u8 pixel_width, fmt;
 	u16 read_val;
 
 	rxmsa_mvid = xdprxss_read(state, XDPRX_MSA_MVID_REG);
@@ -482,7 +481,6 @@ static int xdprxss_get_stream_properties(struct xdprxss_state *state)
 	read_val = xdprxss_read(state, XDPRX_DTG_REG);
 	xdprxss_write(state, XDPRX_DTG_REG, (read_val | 0x1));
 	fmt = FIELD_GET(XDPRX_MSA_FMT_MASK, rxmsa_misc);
-	rxmsa_bpc = FIELD_GET(XDPRX_MSA_BPC_MASK, rxmsa_misc);
 
 	switch (fmt) {
 	case XDPRX_COLOR_FORMAT_422:
-- 
2.31.1

